frontend/src/app/user/user.service.ts

```diff
@@ -59,6 +64,7 @@ export class UserService {
     localStorage.removeItem('userName');
 
     this.checkUserStatus();
+    this.router.navigate(["/"])
```

Comment:
It could also make sense to implement this in the controller that uses this service. Advantage: you could implement different redirections depending on where a user clicks on a logout button. This way, it will always redirect to start page, wherever you use this service method

This is just an FYI, no changes need to be made. It's a matter of frontend architecture